docs/training/apis-howto/api-pytorch-ug.rst

```diff
@@ -638,7 +636,271 @@ Take the following code for example.
+   +       latest_checkpoint = None
+   +   else:
+   +       # On-cluster: Determined will automatically detect distributed context.
+   +       distributed_context = None
+   +       # On-cluster: configure the latest checkpoint for pause/resume training functionality.
+   +       latest_checkpoint = det.get_cluster_info().latest_checkpoint
+
+   +     with det.pytorch.init(
+   +       distributed=distributed_context
+         ) as train_context:
+             trial = MNistTrial(train_context)
+             trainer = det.pytorch.Trainer(trial, train_context)
+             trainer.fit(
+                 max_length=pytorch.Epoch(1),
+                 checkpoint_period=pytorch.Batch(100),
+                 validation_period=pytorch.Batch(100),
+   +             latest_checkpoint=latest_checkpoint,
+             )
+
+To run Trainer API solely on-cluster, the code is much simpler:
+
+.. code:: python
+
+   def main():
+       with det.pytorch.init() as train_context:
+           trial_inst = model.MNistTrial(train_context)
+           trainer = det.pytorch.Trainer(trial_inst, train_context)
+           trainer.fit(
+               checkpoint_period=pytorch.Batch(100),
+               validation_period=pytorch.Batch(100),
+               latest_checkpoint=det.get_cluster_info().latest_checkpoint,
+           )
+
+Submit Your Trial for Training on Cluster
+=========================================
+
+To run your experiment on cluster, you'll need to create an experiment configuration (YAML) file.
+Your experiment configuration file must contain searcher configuration and entrypoint.
+
+.. note::
+
+   ``hyperparameters.global_batch_size`` is required if ``max_length`` is configured in records
+
+.. code:: python
+
+   name: pytorch_trainer_trial
+   searcher:
+     name: single
+     metric: validation_loss
+     max_length:
+       epochs: 1
+   resources:
+     slots_per_trial: 8
+   entrypoint: python3 -m determined.launch.torch_distributed python3 train.py
+
+Submit the trial to the cluster:
+
+.. code:: bash
+
+   det e create det.yaml .
+
+If your training code needs to read some values from the experiment configuration,
+``pytorch.init()`` accepts an ``exp_conf`` argument which allows calling
+``context.get_experiment_config()`` from ``PyTorchTrialContext``.
+
+Loading Checkpoints
+===================
+
+To load a checkpoint from a checkpoint saved using Trainer, you'll need to download the checkpoint
+to a file directory and use an import helper method to import modules. You should instantiate your
+loaded Trial with a ``CheckpointLoadContext``.
+
+``det.import_from_path`` allows you to import from a specific directory and cleans up afterwards.
+Even if you are importing identically-named files, you can import them as separate modules. This is
+intended to help when you have, for example, a current model_def.py, but also import an older
+model_def.py from a checkpoint into the same interpreter, without conflicts (so long as you import
```

Comment:
`model_def.py` needs double backticks I think.  Perhaps in the docstring I wrote too.